caluma/caluma_form/serializers.py

```diff
@@ -197,6 +197,7 @@ class Meta:
 
 
 class SaveTextQuestionSerializer(SaveQuestionSerializer):
+    min_length = IntegerField(min_value=1, required=False, allow_null=True)
```

Comment:
Shouldn't `min_value` be 0? Otherwise, you'd need to explicitly set it to `None`/`null` when you don't want to enforce minimum length, leading to additional check required in the frontend

Reply:
Sure? Shouldn't `required=False, allow_null=True` allow for that case? See also how `max_length` is defined on the line below.

Reply:
Yes, it would allow setting it to `null` but not to 0, which in my opinion should be allowed (and cause the same effect)

Reply:
Ah, now I understand what you mean. The additional check in the frontend is already implemented, you can try it with `maxLength` here: https://demo.caluma.io/form-builder/rst/questions/test

Reply:
Hmm, `maxLength` has a bit other semantics, as a max length of 0 doesn't make sense. Setting `minLength` to 0 however could make sense from a user's perspective, even if it's the same as the value being absent.

However I don't have strong enough feelings about this to be pedantic about it, so feel free to merge :)